### PR TITLE
dns: cleanup and update mkcloud's crowbar entry in /etc/hosts

### PR DESCRIPTION
**Pull request by MaximilianMeister**

### CODE REVIEW

File: scripts/mkcloud [hunk 1]
```diff
@@ -279,6 +279,10 @@ function cleanup()
         vgchange -an $cloudvg
         dd if=/dev/zero of=$cloudpv count=1000
     fi
+
+    # 3. cleanup file entries
+    sed -i '/ crowbar.* crowbar$/d' /etc/hosts
```
Comment (tboerger) [inline scripts/mkcloud]: That's too generic, that will cleanup my custom hosts entries :(

Comment (rsalevsky) [inline scripts/mkcloud]: Yep, I destroyed already my host file.

Comment (MaximilianMeister) [inline scripts/mkcloud]: >  Yep, I destroyed already my host file.

try without -i next time

**Comment by bmwiedemann** [comment]
:-1: I would leave the old code as it could break hosts with multiple clouds. And if you have just one cloud, you dont often need to change your admin net and mkcloud itself always uses $adminip and never the crowbar name - it is just for convenience for the developer.

**Comment by MaximilianMeister** [comment]
@bmwiedemann okay. thanks for the explanation. will close this then

**Comment by tboerger** [comment]
I like the functionality, but it needs to be more specific :)

**Comment by MaximilianMeister** [comment]
@tboerger the regex?

i have designed the regex not to destroy `mkcloud.cloud.suse.de` but if everyone has his own manual entries in a different way, then it will be difficult to find the right pattern :)

File: scripts/mkcloud [hunk 2]
```diff
@@ -408,7 +412,7 @@ function onhost_deploy_image()
 
 function onhost_add_etchosts_entries()
 {
-    grep -q crowbar /etc/hosts || echo "$adminip crowbar.$cloudfqdn crowbar" >> /etc/hosts
+    echo "$adminip crowbar.$cloudfqdn crowbar" >> /etc/hosts
```
Comment (tboerger) [inline scripts/mkcloud]: maybe ```grep -q "$adminip.$cloudfqdn crowbar" /etc/hosts || echo "$adminip crowbar.$cloudfqdn crowbar" >> /etc/hosts```?

File: scripts/mkcloud (hunk 1 above)
Comment (tboerger) [inline scripts/mkcloud]: ```sed -i -e "s/$adminip crowbar.$cloudfqdn crowbar/d" /etc/hosts```? or something similar?

**Pull request closed without merging**